opm/input/eclipse/Schedule/Schedule.hpp

```diff
@@ -224,7 +224,7 @@ namespace Opm {
         std::vector<std::string> wellNames(const std::string& pattern) const;
         std::vector<std::string> wellNames(std::size_t timeStep) const;
         std::vector<std::string> wellNames() const;
-
+        //const ScheduleGrid& get_schedulegrid() const{ return this->grid; };
```

Comment:
Please don't add these kinds of commented-out statements.  The function is either needed, in which case it should be uncommented, or it is not needed in which case it should not be present.